js/adapt-mathJax.js

```diff
@@ -1,20 +1,50 @@
 define([ "core/js/adapt" ], function(Adapt) {
 
+	function loadScript(scriptObject, callback) {
+		var head = document.getElementsByTagName('head')[0];
+		var script = document.createElement('script');
+
+		script.type = scriptObject.type || 'text/javascript';
+    
+		if (scriptObject.src) {
+			script.src = scriptObject.src;
+		}
+
+		if (scriptObject.text) {
+			script.text = scriptObject.text;
+		}
+    
+		if (callback) {
+			// Then bind the event to the callback function.
+			// There are several events for cross browser compatibility.
+			script.onreadystatechange = callback;
+			script.onload = callback;
+		}
+    
+        // Append the <script> tag.
+        head.appendChild(script);
+	}
+
 	function setUpMathJax() {
+		Adapt.trigger('plugin:beginWait');

```

Comment:
does this create a dependency on Framework [v2.0.16](https://github.com/adaptlearning/adapt_framework/releases/tag/v2.0.16) or would it still work OK if you had an earlier version?